modules/redisearch.ts

```diff
@@ -175,54 +175,60 @@ export class Redisearch extends Module {
     async aggregate(index: string, query: string, parameters?: FTAggregateParameters): Promise<number> {
         let args: string[] = [index, query];
         if(parameters !== undefined) {
-            if(parameters.load !== undefined) {
-                args.push('LOAD')
-                if(parameters.load.nargs !== undefined)
+            if (parameters.load !== undefined) {
```

Comment:
I don't see a reason to add spaces after the if before brackets while all the code is done without it.  Just creates inconsistency in the code.